### DISCUSSION TITLE
i cant update to the new version

### DISCUSSION THREAD

**Discussion by NitinNR** [chatwoot/chatwoot#3601] [General]
I got no error while installing the new version.it was seems all ok. Please tell me how to check the other logs other than journactl one

following the below procedure
https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm

using ubuntu for deployment

**Comment by sojan-official** [comment]
did you follow the steps described here : https://www.chatwoot.com/docs/self-hosted/deployment/linux-vm#upgrading-to-a-newer-version-of-chatwoot ?

**Reply by NitinNR** [reply]
Yes.

**Comment by vishnu-narayanan** [comment]
Could you cd into your chatwoot folder and share your commit id?

```
cd chatwoot
git branch
git log
```

v2.0.0 commit id should be `a7623041bf9b2a89841da0cec0985738629b8db6`

https://github.com/chatwoot/chatwoot/commits/master
